keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -487,7 +487,8 @@ Here is the workflow:
 
 This workflow describes the automation designed for deleting a `Bucket` instance and optionally the related backend bucket. Revoking access to this bucket is covered in [Revoke Bucket Access](#revoke-bucket-access).
 
-> Note: it is important to understand that when a `BucketRequest` deletion triggers the delete of the associated `Bucket` instance, the `Bucket` instance is deleted regardless of access. This means that even if a `BucketAccess` references a `Bucket`, the `Bucket is still deleted and the BA is orphaned. Additionally, even if multiple `Bucket` instances point to the same backend bucket, if the retention policy is "Delete" then the bucket is deleted and workloads may fail. Potentially, this may only apply to MVP.
+> Note: it is important to understand that when a `BucketRequest` deletion triggers the delete of the associated `Bucket` instance, the `Bucket` instance is deleted regardless of access. This means that even if a `BucketAccess` references a `Bucket`, the `Bucket is still deleted and the BA is orphaned. Additionally, even if multiple `Bucket` instances point to the same backend bucket, if the retention policy is "Delete" then the bucket is deleted and workloads may fail. Potentially, this may only apply to MVP. 
+A `BucketAccess` instance is deleted only once the underlying Pod stop using the access credentials. Additionally, even if multiple `Bucket` instances point to the same backend bucket, if the retention policy is "Delete" then the bucket is deleted and workloads may fail. Potentially, this may only apply to MVP. 
```

Comment:
The second sentence is a repeat of above, and:

 "A `BucketAccess` instance is deleted only once the underlying Pod stop using the access credentials"
Is this the behavior we want? Meaning, a BA will be deleted by cosi even if its BAR exists.

Reply:
if user tries to delete BA by deleting BAR or directly it will not be deleted until Pod goes away.

Reply:
So there are 2 delete triggers: 1) user deletes BAR, but if pod still accessing BA then delete is deferred. 2) pod terminates, but if BAR/BA still exist then delete is deferred. 
So the _only_ way to delete a BA (unless admin does it) is for the BAR to be deleted _and_ the app pod to terminate. Is this they way you see it?

Reply:
yes, I agree. should we spell out these two?